tests/ops/test_helpers/saas_test_utils.py

```diff
@@ -14,12 +14,15 @@ def poll_for_existence(
     retries: int = 10,
     interval: int = 5,
     existence_desired=True,
+    verification_count=1,
```

Comment:
Added a `verification_count` to make sure we get a certain number of successes in a row before considering the data available. There are some APIs that don't return data on requests made immediately after an initial success.